@sticknologic please help | Add CNAME record for www SUBDOMAIN at ayaskant007.is-a.dev FOR NETLIFY, my A record PR was approved.

ayaskant007 opened the pull request:
Added a CNAME record pointing 'www' to 'portfolio-007a.netlify.app.' in the ayaskant007 domain configuration.
 @sticknologic please help

# Requirements

- [x] I **agree** to the [Terms of Service](https://is-a.dev/terms). <!-- Your request MUST follow the TOS to be approved. -->
- [x] My file is following the [domain structure](https://docs.is-a.dev/domain-structure/). <!-- Your JSON file is in the domains directory, the name is valid, etc. -->
- [x] My website is **reachable** and **completed**. <!-- We do not permit simple "Hello, world!" or simply copied template websites with minimal changes. -->
- [x] My website is **software development** related. <!-- Only your root subdomain needs to meet this requirement. -->
- [x] My website is **not for commercial use**. <!-- Your website's purpose should not be to generate any form of revenue or profit. -->
- [x] I have provided contact information in the `owner` key. <!-- Provide your email in the `email` field or another platform (e.g. X/Twitter or Discord) for contact. -->
- [x] I have provided a preview of my website below. <!-- This step is required for your PR to be approved. -->

# Website Preview
https://portfolio-007a.netlify.app.
<img width="1912" height="994" alt="screenshot-1764507424207" src="https://github.com/user-attachments/assets/239172b5-9004-4a04-8a55-49831e048fa0" />

#CNAME RECORD REQUIREMENT
<img width="760" height="615" alt="image" src="https://github.com/user-attachments/assets/3e94f8dd-852d-41a0-bdc3-efde983ac482" />

ayaskant007 committed:
Add CNAME record for www subdomain

Added a CNAME record pointing 'www' to 'portfolio-007a.netlify.app.' in the ayaskant007 domain configuration.

ayaskant007 committed:
Merge branch 'main' into main

iostpa commented on domains/ayaskant007.json:
```diff
@@ -4,6 +4,7 @@
     "records": {
-        "A": ["75.2.60.5"]
+        "A": ["75.2.60.5"],
+        "CNAME": ["www", "portfolio-007a.netlify.app."]
```
You need to make a separate file for the www nested subdomain (or sub-subdomain).

ayaskant007 committed:
Move CNAME record to www.ayaskant007.json

The CNAME record for 'www' was removed from ayaskant007.json and added to a new file, www.ayaskant007.json, to separate root and subdomain DNS records.

ayaskant007 committed:
Merge branch 'main' of https://github.com/ayaskant007/register

ayaskant007 committed:
Remove trailing dot from CNAME record

Updated the CNAME record for 'www' in www.ayaskant007.json to remove the trailing dot from the target domain. This ensures proper DNS configuration and compatibility with some DNS providers.

ayaskant007 committed:
Update CNAME record and add record details field

Moved 'www' from the CNAME array to a new 'record details' field under owner, and updated the CNAME record to only include the target domain.

ayaskant007 committed:
Update CNAME record format in domain config

Changed the CNAME record from an array to a string in www.ayaskant007.json to standardize the record format.

ayaskant007 committed:
Remove redundant record details from domain config

Deleted the 'record details' field from the owner object in www.ayaskant007.json to avoid duplication, as CNAME is already specified in the records section.

ayaskant007 committed:
Merge branch 'main' into main

ayaskant007 committed:
Remove trailing comma in domain config

Eliminated a trailing comma from the 'owner' object in www.ayaskant007.json to ensure proper JSON formatting.

ayaskant007 committed:
Merge branch 'main' of https://github.com/ayaskant007/register
